zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -81,7 +81,12 @@ public void open(final Controller controller) {
     this.controller = controller;
     clientAdapter = ClientAdapter.of(configuration);
     final var searchEngineClient = clientAdapter.getSearchEngineClient();
-    final var schemaManager = clientAdapter.createSchemaManager(provider);
+    final var schemaManager =
+        new SchemaManager(
+            searchEngineClient,
+            provider.getIndexDescriptors(),
+            provider.getIndexTemplateDescriptors(),
+            configuration);
 
     schemaStartup(schemaManager, searchEngineClient);
```

Comment:
🔧 another non-related suggestion/question: could this `schemaStartup` be part of the schema manager class?
like `SchemaManager.startup()` 
I see that it doesn't need anything from the exporter that it doesn't have access too already.

Reply:
sounds like a good idea